rocketpool-cli/commands/wallet/rebuild.go

```diff
@@ -54,41 +54,52 @@ func rebuildWallet(c *cli.Context) error {
 		}(customKeyPasswordFile)
 	}
 
+	var enablePartialRebuildValue = false
+	if enablePartialRebuild.Name != "" {
+		enablePartialRebuildValue = c.Bool(enablePartialRebuild.Name)
+	}
+
 	// Log
 	fmt.Println("Rebuilding node validator keystores...")
+	fmt.Printf("Partial rebuild enabled: %s.\n", enablePartialRebuild.Name)
 
 	// Rebuild wallet
-	response, err := rp.Api.Wallet.Rebuild()
-	if err != nil {
-		return err
+	response, _ := rp.Api.Wallet.Rebuild(enablePartialRebuildValue)
```

Comment:
we still need to catch and return the error here, as there can be http failures between the cli and the api